app/modules/recordings.ts

```diff
@@ -1,5 +1,5 @@
 import {PopUpState, Action} from '../types'
-const INITIAL_POPUP_STATE : PopUpState = { enabled: false , errorMessage: "",requests: []}
+const INITIAL_POPUP_STATE : PopUpState = { enabled: false , errorMessage: "", requests: [], selectedReqIds: [], checked : false}
```

Comment:
As discussed, why this `checked` is needed? please remove

Reply:
Tried removing it, we later check the value of checked below as 
https://github.com/code-mancers/interceptor/blob/0c253454b3d57d02aa2e7abfee18d50d2f1a573e/app/modules/recordings.ts#L23

If removed, the ID's wont get added to the array